streamer/src/nonblocking/quic.rs

```diff
@@ -1193,7 +1203,7 @@ async fn handle_chunk(
                 // tho, in which case we drop the stream.
```

Comment:
```suggestion
                // tho, in which case we report error.
```

Because later this error leads to dropping the whole connection.

Reply:
done